### ISSUE TITLE
Add `-c` argument

### ISSUE THREAD

**Issue by piranna** [piranna/easy-coveralls#1]
Add an argument to exec a specific command instead of only the scripts defined on `package.json`.

**Comment by kristianmandrup** [comment]
Yeah, exactly! My package `main` file points to `dist/bundle.js` generated by webpack. Not exactly what I want to do coverage for. Cheers!

**Comment by piranna** [comment]
I'm a bit busy latelly, but pull-requests are welcome :-)
